HPXMLtoOpenStudio/measure.rb

```diff
@@ -181,7 +175,8 @@ def self.create(hpxml, runner, model, hpxml_path, epw_path, weather, output_dir,
     @schedules_file = SchedulesFile.new(runner: runner, model: model,
                                         schedules_paths: @hpxml.header.schedules_filepaths,
                                         year: Location.get_sim_calendar_year(@hpxml.header.sim_calendar_year, epw_file),
-                                        unavailable_periods: @hpxml.header.unavailable_periods)
+                                        unavailable_periods: @hpxml.header.unavailable_periods,
+                                        output_path: File.join(output_dir, 'in.schedules.csv'))
```

Comment:
Use the "in" to differentiate from schedules, e.g., reported by energyplus?

Reply:
Just thought it made sense to be consistent with in.epw, in.xml, in.idf, in.osm, etc. This is the file that will be referenced by in.idf.